tests/test_plotting.py

```diff
@@ -0,0 +1,41 @@
+import shutil
+from pathlib import Path
+from nesta_ds_utils import plotting
```

Comment:
I think you and @emily-bicks are doing imports for tests a bit differently. Maybe check and confirm one way.

Reply:
Done